concepts/functions/about.md

```diff
@@ -170,49 +201,57 @@ Now, we can call the functions in the order we want.
 
 ## Scope of Variables
 
-If variable is defined inside a function, then it will be only accessible inside the function. If we want to access the variable outside the function, we need to use the [`global`][global] keyword. [`nonlocal`][nonlocal] keyword is used to access the variable inside a nested function.
+If variable is defined inside a function, then it will be only accessible inside the function.
+If we want to access the variable name outside the function scope (_or namespace_), we need to use the [`global`][global] keyword.
+[`nonlocal`][nonlocal] keyword is used to access the variable inside a nested function.
 
 ```python
->>> x = 30
+>>> num = 30
 >>> def random_function():
-        x = 10
-        print('Inside function:', x)
+        num = 10
+        print('Inside function:', num)
 ```
 
-As `x` is defined inside the `random_function`, it is limited to the scope of the `random_function` only. Calling the function will not alter the value of the variable outside the function.
+As `num` is defined inside the `random_function`, it is limited to the scope of the `random_function` only.
+Calling the function will not alter the value of the variable outside the function.
 
 ```python
->>> x = 30
+>>> num = 30
 
-# regardless of whether we call the function or not, the value of x will be 30
+# regardless of whether we call the function or not, the value of num will be 30
 >>> random_function()
 Inside function: 10
->>> x
+>>> num
 30
 ```
 
 We can access the variable inside the outer function using the `global` keyword.
 
 ```python
->>> x = 30
+>>> num = 30
 >>> def random_function():
-        global x
-        x = 10
-        print('Inside function:', x)
+        global num
+        num = 10
+        print('Inside function:', num)
 ```
 
-As we have used the `global` keyword, the value of `x` will be changed.
+As we have used the `global` keyword, the value of `num` will be changed.
 
 ```python
 >>> random_function()
 Inside function: 10
->>> x
+>>> num
 10
 ```
 
 ## Functions as first class objects
 
-In python, functions can be assigned to variables and passed as arguments to other functions. They can be used as return values. Functions can also be placed into a sequence([`list`][list], [`tuple`][tuple] etc) or as value in a [`dict`][dict]. Functions can be used anywhere than any other object can be used. This is because _functions are [`first class objects`][first class objects]_.
+In python, functions can be assigned to variables and passed as arguments to other functions.
+They can be used as return values.
+Functions can also be placed into a sequence([`list`][list], [`tuple`][tuple] etc) or as value in a [`dict`][dict].
+Functions can be used anywhere any other object can be used.
+This is because _functions are [`first class objects`][first class objects]_.
```

Comment:
You're right. I think the intent is to indicate that it is a link. It's hard to tell the blue link from regular black text. It would be nice if there was a set way we used to indicate a link that is more noticeable.